util.c

```diff
@@ -52,6 +52,10 @@
 #include <ykdef.h>
 #endif /* HAVE_CR */
 
+#if HAVE_MYSQL
```

Comment:
please use #ifdef here

Reply:
should I just replace #if with #ifdef?

Reply:
Yes, after dropping setting HAVE_MYSQL to 0 in the case where we don't have it

Reply:
done